src/sbin/lorax

```diff
@@ -302,6 +325,24 @@ def get_dnf_base_object(installroot, repositories, mirrorlists=None,
             log.error("Error fetching metadata for %s: %s", repo_name, e)
             return None
 
+    # Enable repos listed on the cmdline
+    for r in enablerepos:
+        repolist = dnfbase.repos.get_matching(r)
+        if not r:
+            log.warning("%s is an unknown repo, not enabling it", r)
+        else:
+            log.info("Enabled repo %s", r)
+            repolist.enable()
```

Comment:
I think the above two lines should be swapped. Otherwise, one could e.g. get a traceback from ``repolist.enable()`` with *Enabled repo something* already in the log. The same applies to the other for cycle below.